### PR TITLE
add-doc-fix52592

### PR DESCRIPTION
**Pull request by dbsid**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add doc for the new fix control 52592

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: optimizer-fix-controls.md [hunk 1]
```diff
@@ -26,6 +26,15 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 
 ## Optimizer Fix Controls 参考
 
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">从 v8.4.0 版本开始引入</span>
+
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 此变量控制是否允许 FastPlan 计划。如果设置为 `ON`，优化器将回退到 Coprocessor，Get 和 BatchGet 都将被禁用。只有当 FastPlan 比 Coprocessor 更昂贵时才应设置为 `ON`。 FastPlan 不支持列投影，以下是需要列投影且 FastPlan 被禁用的常见情况：
```
Comment (songrijie) [inline optimizer-fix-controls.md:33]:
```suggestion
- 控制是否允许通过 Get 或 BatchGet 获取数据。如果设置为 `ON`，优化器会强制选择 Coprocessor，而 Get 和 BatchGet 都将被禁用。 Get 和 BatchGet 不支持列投影，在部分场景中执行代价比 Coprocessor 请求高，设置为 `ON` 会获得更好的性能。以下是推荐强制选择 Coprocessor 的常见情况：
```
我们文档里似乎没有 FastPlan 这个概念。

File: optimizer-fix-controls.md [hunk 2]
```diff
@@ -26,6 +26,15 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 
 ## Optimizer Fix Controls 参考
 
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">从 v8.4.0 版本开始引入</span>
+
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 控制是否允许通过 Get 或 BatchGet 获取数据。如果设置为 `ON`，优化器会强制选择 Coprocessor，而 Get 和 BatchGet 都将被禁用。 Get 和 BatchGet 不支持列投影，在部分场景中执行代价比 Coprocessor 请求高，设置为 `ON` 会获得更好的性能。以下是推荐强制选择 Coprocessor 的常见情况：
+	- 具有多列的宽表，应用程序只查询少量列子集。
+	- 包含大型 JSON 值的 JSON 列的表，查询要么不检索 JSON 列，要么只提取 JSON 内容的小部分。
```
Comment (winoros) [inline optimizer-fix-controls.md:35]: 因为 Get 目前会涉及节省网络还是高 qps，所以也写一下吧

File: optimizer-fix-controls.md [hunk 3]
```diff
@@ -71,6 +71,14 @@ SET SESSION tidb_opt_fix_control = '44262:ON,44389:ON';
 - 此开关控制优化器进行启发式访问路径选择的阈值。当某个访问路径（如 `Index_A`）的估算行数远小于其他访问路径时（默认为 `1000` 倍），优化器会跳过代价比较直接选择 `Index_A`。
 - `0` 表示关闭此启发式访问路径选择策略。
 
+### [`52592`](https://github.com/pingcap/tidb/issues/52592) <span class="version-mark">从 v8.4.0 版本开始引入</span>
+
+- 默认值：`OFF`
+- 可选值：`ON`、`OFF`
+- 控制是否允许通过 Get 或 BatchGet 获取数据。如果设置为 `ON`，优化器会强制选择 Coprocessor，而 Get 和 BatchGet 都将被禁用。 Get 和 BatchGet 不支持列投影，在部分场景中执行代价比 Coprocessor 请求高，设置为 `ON` 会获得更好的性能。以下是推荐强制选择 Coprocessor 的常见情况：
+    - 具有多列的宽表，应用程序只查询少量列子集。
+	- 包含大型 JSON 值的 JSON 列的表，查询要么不检索 JSON 列，要么只提取 JSON 内容的小部分。
```
Comment (qiancai) [inline optimizer-fix-controls.md:80]:
```suggestion
- 此开关控制是否禁用 `Point Get` 和 `Batch Point Get` 算子执行查询。默认值 `OFF` 代表允许通过 `Point Get` 和 `Batch Point Get` 执行查询。 如果设置为 `ON`，优化器会禁用 `Point Get` 和 `Batch Point Get`，强制选择 Coprocessor 执行查询。
- `Point Get` 和 `Batch Point Get` 不支持列投影（即无法只返回部分列的数据），这意味着在某些场景中其执行效率可能低于 Coprocessor，此时设置为 `ON` 可以提高查询性能。以下是推荐设置为 `ON` 的场景：

    - 查询具有多列的宽表，且仅涉及表中的少量列。
    - 查询包含大型 JSON 值的表，且不需要检索整个 JSON 列，或仅需提取 JSON 列中的小部分数据。
```

**Review by Oreoxmt — APPROVED** [review]

**Review by winoros — APPROVED** [review]

**Review by songrijie — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**